pkg/oc/cli/debug/debug.go

```diff
@@ -345,11 +348,15 @@ func (o *DebugOptions) RunDebug() error {
 		ObjectMeta: template.ObjectMeta,
 		Spec:       template.Spec,
 	}
-	pod.Name, pod.Namespace = fmt.Sprintf("%s-debug", generateapp.MakeSimpleName(infos[0].Name)), infos[0].Namespace
+	ns := infos[0].Namespace
```

Comment:
Why is this needed, I'd expect every info object to have a namespace unless that's a non-namespaced one (like node) but in that case empty namespace is still fine

Reply:
It's not, because we have to create a pod with a namespace, or the create call will fail.  `o.Namespace` is the default.